### ISSUE TITLE
Updating filecache after creation of file

### ISSUE THREAD

**Issue by e-alfred** [Aypac/files_latexeditor#4]
Hello,

as a workaround for the filecache issue one could set filesystem_check_changes to 1 in config.php. Here is more info on this: https://doc.owncloud.org/server/8.2/admin_manual/configuration_server/config_sample_php_parameters.html

Hopefully the bug can be resolved soon.

**Comment by DrSAR** [comment]
If I understood this right, the scanning happens (is supposed to happen) in 
https://github.com/Aypac/files_latexeditor/blob/master/js/latexeditor.js#L168

```
// Update OwnCloud File Chache via files API
console.log('Scanning files in '+path);
scanFiles(true, path, ''); 
```

I don't know where scanFiles is coming from. 

However, it seems that [owncloud is providing useful hooks](https://doc.owncloud.org/server/9.0/developer_manual/app/hooks.html) for this though, e.g. scanFolder

**Comment by e-alfred** [comment]
@darksniper94 Maybe this could help you solve the problem?

**Comment by darksniper94** [comment]
Correct, this was the function for scanning the directory for new files created after latex compile.
Unfortunately, the function discovered new files (info output was seen in browser console), but didn't referenced them in the database, so the folder didn't contain the new files.

 I think , I try the function scanFolder() from owncloud hooks, as DrSAR sad.
